{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/frontend-editing/index.js

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios'
+
+import LazyImage from '../lazy-images/lazy-image'
+import { ImageEditable, SimpleEditable, WYSIWYGEditable } from './editable'
+
+export function setupEditing () {
+  const editables = document.querySelectorAll('.js-SimpleEditable')
+  for (const el of editables) {
+    new SimpleEditable({ el })
+  }
+
+  const WYSIWYGs = document.querySelectorAll('.js-WYSIWYGEditable')
+  for (const el of WYSIWYGs) {
+    new WYSIWYGEditable({ el })
+  }
+
+  const images = Array.from(document.querySelectorAll('.js-ImageEditable')).map(el => new ImageEditable({ el }))
+
+  window.addEventListener('message', (e) => {
+    if (e.origin === 'http://localhost:3000') {
```

Comment:
I am not sure of the purpose of this.

Reply:
Scratch that, I see it. It'd make more sense to check against `window.location.origin`. As elsewhere, the simpler, negative condition (event origin != the document's origin) should be checked against with an immediate exit rather than wrapping the entirely of the common case in the if clause.